packages/calcite-components/src/components/combobox/combobox.e2e.ts

```diff
@@ -1120,14 +1120,118 @@ describe("calcite-combobox", () => {
         chips = await page.findAll("#myCombobox >>> calcite-chip");
         expect(chips.length).toEqual(2);
       });
+    });
+  });
 
-      it("should delete last chip on Delete", async () => {
-        await element.click();
+  describe("deleting items with the keyboard", () => {
```

Comment:
I think this can go in the above describe block since it also falls within `keyboard interaction`.